cypress/e2e/filter/filterCases.cy.ts

```diff
@@ -43,7 +43,7 @@ describe("Case list", () => {
         users: [
           {
             username: "Bichard01",
-            visibleForces: ["011111"],
+            visibleForces: ["0011111"],
```

Comment:
Added leading 0 because previously, the transformer function in the User entity trimmed the leading 0 from force codes when reading from the DB. With this PR we only trim the leading 0 from the codes when checking the case visibility [just like old Bichard](https://github.com/ministryofjustice/bichard7-next/blob/main/bichard-backend/src/main/java/uk/gov/ocjr/mtu/br7/errorlistmanager/data/RecordSelectionSQLGenerator.java#L704).